emergence_lib/src/graphics/ui/hover_panel.rs

```diff
@@ -0,0 +1,180 @@
+//! Create and update a panel to display info about the hovered tile.
+use bevy::prelude::*;
+
+use crate::{
+    cursor::CursorTilePos,
+    organisms::{organism_details::HoverDetails, structures::crafting::CraftingState},
+};
+
+use super::RightPanel;
+
+/// The panel to display information on hover.
+#[derive(Debug, Component)]
+pub struct HoverPanel;
+
+/// The text to display the position of the tile.
+#[derive(Debug, Component)]
+pub struct PositionText;
+
+/// The text to display the type of organism on the tile.
+#[derive(Debug, Component)]
+pub struct OrganismText;
+
+/// The text for all details regarding crafting.
+#[derive(Debug, Component)]
+pub struct CraftingText;
+
+/// Create the hover panel in the UI.
+pub fn setup_hover_panel(
+    mut commands: Commands,
+    asset_server: Res<AssetServer>,
+    query: Query<Entity, With<RightPanel>>,
+) {
+    let key_text_style = TextStyle {
+        color: Color::rgb(0.9, 0.9, 0.9),
+        font: asset_server.load("fonts/FiraSans-Medium.ttf"),
```

Comment:
We probably don't want to be reloading this constantly: we should store a handle and pass out weak copies.

Reply:
The font handles are now cached in a resource to use throughout the UI.